backend/tarsy/auth/dependencies.py

```
@@ -0,0 +1,115 @@
+
+logger = logging.getLogger(__name__)
+
+
+def get_jwt_service() -> JWTService:
+    """Get JWT service instance configured with current settings."""
+    return JWTService(get_settings())
+
+
+async def verify_jwt_token(
+    request: Request,
+    token: Optional[HTTPAuthorizationCredentials] = Depends(security),
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)